<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editionMenuClass</class>
 <widget class="QWidget" name="editionMenuClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_Back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_Parcours">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>571</width>
     <height>211</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Num</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nom</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Difficulté</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Date</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_Titre">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>311</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Édition</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Add">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>130</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Delete">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>130</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
